pages/common/ocaml.md

```diff
@@ -2,6 +2,7 @@
 
 > The OCaml repl (read-evaluate-print-loop).
 > Interprets Ocaml commands.
+> More information: <https://ocaml.org/>.
```

Comment:
`curl https://ocaml.org` and `curl https://ocaml.org/` is the same HTML, could you please remove the `/` in the pages?
```suggestion
> More information: <https://ocaml.org>.
```

Reply:
Done.